data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Bactrian Unlock [0]"
+	minor
+	name "Locate Missing Scientists"
+	description "A Deep Star Queen transporting top scientists has gone missing. It was last seen near the Betelgeuse system in the Far North. Find its whereabouts and return it to <destination> if possible."
+	source
+		attributes deep
+	destination Valhalla
+	waypoint Betelgeuse
+	to offer
+		has "Smuggler's Den, Part 2: done"
+		has "Expedition to Hope 2: done"
+		has "WR Star 1: done"
+		has "Rescure Miners 2: done"
+		has "Humanitarian 2: done"
+		has "Terraforming 5: done"
+	on offer
+		conversation
+			`While you are watching one of the telescreens in a restaurant located in the spaceport, you are approached by a man and a woman, both wearing Deep Security uniforms.`
+			`	"Hello. Are you <first> <last>?" the man asks`
+			choice
+				`	"Yes I am. Is there something you need?"`
+					goto named
+				`	"Sorry, but I'm not the person you're looking for."`
+				
+			`	The man looks a little embarrassed after you say that. "Alright. Sorry for bothering you." The two make their way out of the restaurant and you return to watching the telescreen.`
+				decline
```

Comment:
@Amazinite : can be solved by adding a deadline upon defer (e.g. event "Bactrian unlock timeout" 2) and adding this into the `to offer` with a has not event...
In this case, the player could only return the next day (but not the 2nd next day, since the event would be active by then).
But hey... the entire story does not have any deadlines yet, so who cares if you defer for 30 days? And: that Star Queen is there, 1 month later.

You start in the Deep. Let's take Valhalla for simplicity.
Distance to Betelgeuse, first Waypoint: 9 Hyperdrive jumps, or 6 jumps with Jump Drive.
You do not proceed to Arneb and do not locate it directly (would be 6 more jumps) but instead you return directly to:
Valhalla. Another 9 (or 6) jumps.
Landing. +1 day, picking up the Faraday escort.
Next stopovers: Memory and Farpoint. 
Memory: 2 jumps from Valhalla. +1 day for landing.
Farpoint: 9 (or 6) jumps from Memory. +1 day for landing.
Proceed to Haven: 4 jumps from Farpoint. +1 day for landing.
You got her - now just return at any slow speed with as many stopovers as you want, just dont get killed. Using regular Jumpdrives you have already spent 5 weeks until you and the Corvette arrive at Haven and can secure here. No wonder that many people died before you could rescue them.